modvendor/modvendor_windows.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Mineiros GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build windows
+
+package modvendor
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/mineiros-io/terramate/project"
+	"github.com/mineiros-io/terramate/tf"
+)
+
+// targetPathDir returns the target path of the module for Windows systems.
+// On Windows, the colon (:) is prohibited in path components other than volume,
+// then it needs to be replaced by something else when vendoring file:// deps.
+func targetPathDir(vendorDir project.Path, modsrc tf.Source) project.Path {
+	tpath := modsrc.Path
+	if modsrc.PathScheme == "file" {

```

Comment:
Just a remainder that from a previous session we are just going to fix the drive name from `C://` to `/C/`.